templates/server/puppetserver/conf.d/auth.conf.erb

```diff
@@ -361,6 +361,17 @@ authorization: {
             name: "puppetlabs experimental"
         },
 <%- end -%>
+<%- if @jolokia_allow_unauthenticated -%>
+        {
+            match-request: {
+                path: "/metrics/v2"
+                type: path
+            }
+            allow-unauthenticated: true
+            sort-order: 500
+            name: "jolokia metrics"
+        },
+<%- else -%>
 <%- unless @jolokia_metrics_allowlist.empty? -%>
```

Comment:
Should this be `elsif !@jolokia_metrics_allowlist.empty?` because to me it doesn't make a lot of sense to allow unauthenticated and a set of specific allows. Especially since they both have the same `name` and `sort-order`.
```suggestion
<%- elsif !@jolokia_metrics_allowlist.empty? -%>
```

And of course fix up the then redundant `end` statement.

Reply:
Replaced `unless` with `elsif` in most recent commit.